scripts/netlifyPackage.js

```diff
@@ -0,0 +1,54 @@
+// @ts-check
+import assert from 'node:assert'
+import { basename, dirname, resolve } from 'node:path'
+import { argv } from 'node:process'
+import { fileURLToPath } from 'node:url'
+import { readFile, writeFile } from 'node:fs/promises'
+
+import execa from 'execa'
+
+const packageJSON = await getPackageJSON()
+
+async function getPackageJSON() {
+  const packageJSONPath = resolve(fileURLToPath(import.meta.url), '../../package.json')
+  const contents = JSON.parse(await readFile(packageJSONPath, 'utf8'))
```

Comment:
nit: We already have a dep on `normalize-package-data` you can use here to avoid `contents` being `any`

There's even a util `src/utils/get-cli-package-json.ts` you should be able to use

Reply:
This use case is a bit different. We're reading `package.json` and then writing it back, so I don't think we should be normalising it.

I have used the `Package` type now.